packages/edit-post/README.md

```diff
@@ -41,61 +41,7 @@ _Parameters_
 
 ### PluginBlockSettingsMenuItem
 
-Renders a new item in the block settings menu.
```

Comment:
It's unfortunate that we don't have the automatic documentation in the editor package's README too but it should be considered separately from this PR.

Reply:
I noticed that, too. It's probably related to the configuration.